.trunk/trunk.yaml

```diff
@@ -0,0 +1,59 @@
+# This file controls the behavior of Trunk: https://docs.trunk.io/cli
+# To learn more about the format of this file, see https://docs.trunk.io/reference/trunk-yaml
+version: 0.1
+cli:
+    version: 1.18.1
+# Trunk provides extensibility via plugins. (https://docs.trunk.io/plugins)
+plugins:
+    sources:
+        - id: trunk
+          ref: v1.4.0
+          uri: https://github.com/trunk-io/plugins
+# Many linters and tools depend on runtimes - configure them here. (https://docs.trunk.io/runtimes)
+runtimes:
+    enabled:
+        - go@1.21.0
+        - node@18.12.1
+        - python@3.10.8
+tools:
+    runtimes:
+        - node
+# This is the section where you manage your linters. (https://docs.trunk.io/check/configuration)
+lint:
+    definitions:
+        - name: eslint
+          commands:
+              - name: lint
+                run: ${workspace}/bin/eslint.js --output-file ${tmpfile} --format json ${target}
+    enabled:
+        - eslint@8.56.0
+        - actionlint@1.6.26
+        - checkov@3.1.38
+        - git-diff-check
+        - markdownlint@0.38.0
+        - oxipng@9.0.0
+        - renovate@37.102.1
+        - shellcheck@0.9.0
+        - shfmt@3.6.0
+        - svgo@3.1.0
+        - trivy@0.48.1
+        - trufflehog@3.63.4
+        - yamllint@1.33.0
+    disabled:
+        - prettier
+    ignore:
+        - linters: [markdownlint]
+          paths:
+              - CHANGELOG.md
+        - linters: [eslint]
+          # Linting of the docs/ javascript code is managed by the lintDocs command.
+          # Default `check` mode will not run on anything in docs
+          paths:
+              - docs/**
+actions:
+    disabled:
+        - trunk-announce
+        - trunk-check-pre-push
+        - trunk-fmt-pre-commit
+    enabled:
+        - trunk-upgrade-available
```

Comment:
Question: Will something like Renovate know to upgrade trunk? Or is this something that needs to happen manually?

Reply:
Currently upgrading is manual - we are working on a PR to make renovate aware of our schema.